wordpress-mobile/WordPress-iOS
# UL&S: update the Continue button behavior in SiteAddressViewController

Ref #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/299

### To test:
1. `rake dependencies`
2. Enable the `.unifiedSiteAddress` feature flag
3. Build & run (preferably on iPhone SE and a larger viewport)
4. Enable the on-screen keyboard if turned off. ( ⌘ + k )
5. Select the textfield. Tap away from the textfield. Observe keyboard behavior.
**Expected:** the pink "Continue" button is anchored to the keyboard and slides up when the keyboard appears. The button slides down when the keyboard is dismissed.

| iPhone 11 Pro screenshots | With keyboard |
| --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/83883017-90415880-a708-11ea-9e16-4e95d2fdf24a.png"><img src="https://user-images.githubusercontent.com/1062444/83883017-90415880-a708-11ea-9e16-4e95d2fdf24a.png" width="350" /></a> | <a href="https://user-images.githubusercontent.com/1062444/83883041-9a635700-a708-11ea-9bd6-4374e052bf4f.png"><img src="https://user-images.githubusercontent.com/1062444/83883041-9a635700-a708-11ea-9bd6-4374e052bf4f.png" width="350" /></a> |

PR submission checklist:
